openmc/mgxs/mgxs.py

```diff
@@ -5257,7 +5271,7 @@ class Chi(MGXS):
     by_nuclide : bool
         If true, computes cross sections for each nuclide in domain
     domain : openmc.Material or openmc.Cell or openmc.Universe or openmc.RegularMesh
-        Domain for spatial homogenization
+        Domain for spatial homogenization`
```

Comment:
```suggestion
        Domain for spatial homogenization
```

Reply:
!! Thank you! I looked at that when looking at my commit in gitk, and didnt see the apostrophe. Assumed I just deleted whitespace. 

Its fixed now. Also, I love the new ability to make changes right here in the github web interface, thats really cool.